contrib/storage-jdbc/src/test/java/org/apache/drill/exec/store/jdbc/TestJdbcPluginWithH2IT.java

```diff
@@ -83,6 +84,7 @@ public void testCrossSourceMultiFragmentJoin() throws Exception {
   }
 
   @Test
+  @Ignore("Ignore this test since h2 mangles dates and times due to improper timezone support.")
```

Comment:
Is it possible to use something like `ExecTest.mockUsDateFormatSymbols()` instead of just ignoring the test?

Reply:
I fixed this by adding explicitly setting the timezone to UFT before these tests.